pkg/grpc/databroker/databroker.go

```diff
@@ -34,27 +37,32 @@ func ApplyOffsetAndLimit(all []*Record, offset, limit int) (records []*Record, t
 	return records, len(all)
 }
 
-// GetAllPages calls GetAll for all pages of data.
-func GetAllPages(ctx context.Context, client DataBrokerServiceClient, in *GetAllRequest) (*GetAllResponse, error) {
-	var res GetAllResponse
-	var pageToken string
+// InitialSync performs a sync with no_wait set to true and then returns all the results.
+func InitialSync(ctx context.Context, client DataBrokerServiceClient, in *SyncRequest) (*SyncResponse, error) {
```

Comment:
Could we replcae the existing test for `GetAllPages` with the new `InitialSync` versino.